qualtricssurvey/private/view.js

```diff
@@ -0,0 +1,11 @@
+function QualtricsSurveyView(runtime, element) {
```

Comment:
@jspayd I think we could leave the build pipeline alone, since as you point out, it doesn't do anything if no .js is found. We may still want to actually remove this JS file (so there's nothing for the build pipeline to find) and update the student_view so that it doesn't add/initialize the JS fragment: CC @caseylitton